build.gradle

```diff
@@ -221,6 +231,20 @@ subprojects {
 		}
 	}
 
+	jar {
+		manifest {
+			attributes(
+				'Built-By': System.properties['user.name'],
+				'Created-By': "${System.properties['java.version']} (${System.properties['java.vendor']} ${System.properties['java.vm.version']})".toString(),
+				'Build-Date': buildDate,
+				'Build-Time': buildTime,
+				'Build-Revision': versioning.info.commit,
+				'Specification-Title': project.name,
+				'Specification-Version': project.version,
```

Comment:
According to the JLS, `project.version` is not appropriate for the `Specification-Version` since `project.version` contains non-digits (e.g., `5.0.0-ALPHA`, `5.0.0-M1`, `5.0.0-SNAPSHOT`, etc.). 

We should therefore use `project.version` for the `Implementation-Version` and a parsed version (with the "-modifier" dropped off) for the `Specification-Version`.

We should also set the `Implementation-Title` and `Implementation-Vendor` for completeness.